docs/quickstart.html

```diff
@@ -32,8 +32,8 @@ <h4 class="anchor-heading">
             the latest Kafka release and extract it:
         </p>
 
-        <pre class="line-numbers"><code class="language-bash">$ tar -xzf kafka_2.13-2.8.0.tgz
-$ cd kafka_2.13-2.8.0</code></pre>
+        <pre class="line-numbers"><code class="language-bash">&gt; tar -xzf kafka_2.13-2.8.0.tgz
```

Comment:
I find the dollar sign more idiomatic for a shell prompt than greater-than.

Reply:
Changed these and all the other occurrences